Scribe/AboutTab/AboutViewController.swift

```diff
@@ -1,42 +1,47 @@
 //
-//  AboutViewController.swift
+//  Copyright (C) 2023 Scribe
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
 import UIKit
+import MessageUI
+import StoreKit
 
-class AboutViewController: UIViewController {
-  @IBOutlet var outerTable: UITableView!
+final class AboutViewController: BaseTableViewController {
 
-  let tableData = AboutTableData.aboutTableData
+  override var dataSet: [ParentTableCellModel] {
+    AboutTableData.aboutTableData
+  }
 
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    title = "About"
-
-    let nib = UINib(nibName: "ParentTableViewCell", bundle: nil)
-    outerTable.register(nib, forCellReuseIdentifier: "ParentTableViewCell")
+    title = NSLocalizedString("about.title", comment: "The title of the about tab")
```

Comment:
Really beautiful stuff, and great that you jumped into already using the naming conventions we're planning on 🙌

Reply:
Thanks 😉 A good thing with this system is that the naming can be changed and it will update the catalog with the new keys (not sure it will move the values accordingly though).